src/content/en/updates/2017/02/media-session.md

```diff
@@ -0,0 +1,445 @@
+which may come from notifications or media keys. Excited? Try out the official
+[Media Session sample]!
+
+<figure>
+  <img src="/web/updates/images/2017/02/tldr.png"
+    alt="Media Session TL;DR;"/>
+</figure>
+
+## Gimme What I Want
+
+You already know all about the Media Session API and are simply coming back to
+copy and paste with no shame some boilerplate code? So here it is.
+
+    if ('mediaSession' in navigator) {
+
+      navigator.mediaSession.metadata = new MediaMetadata({
+        title: 'Never Gonna Give You Up',
+        artist: 'Rick Astley',
+        album: 'Whenever You Need Somebody',
+        artwork: [
+          { src: 'https://dummyimage.com/96x96',   sizes: '96x96',   type: 'image/png' },
+          { src: 'https://dummyimage.com/128x128', sizes: '128x128', type: 'image/png' },
+          { src: 'https://dummyimage.com/192x192', sizes: '192x192', type: 'image/png' },
+          { src: 'https://dummyimage.com/256x256', sizes: '256x256', type: 'image/png' },
+          { src: 'https://dummyimage.com/384x384', sizes: '384x384', type: 'image/png' },
+          { src: 'https://dummyimage.com/512x512', sizes: '512x512', type: 'image/png' },
+        ]
+      });
+
+      navigator.mediaSession.setActionHandler('play', function() {});
+      navigator.mediaSession.setActionHandler('pause', function() {});
+      navigator.mediaSession.setActionHandler('seekbackward', function() {});
+      navigator.mediaSession.setActionHandler('seekforward', function() {});
+      navigator.mediaSession.setActionHandler('previoustrack', function() {});
+      navigator.mediaSession.setActionHandler('nexttrack', function() {});
+    }
+
+## Get into the code
+
+### Let's play 🎷
+
+Add a simple `<audio>` tag to your web page and assign several media sources so
+that the browser can choose which one works best.
+
+    <audio controls>
+      <source src="audio.mp3" type="audio/mp3"/>
+      <source src="audio.ogg" type="audio/ogg"/>
+    </audio>
+
+Note: I could have used a `<video>` tag instead to showcase the Media Session
+API.
+
+As you may know, `autoplay` is disabled for audio elements on Chrome
+for Android which means we have to use the `play()` method of the audio
+element. This method must be triggered by [a user gesture] such as a touch or a
+mouse click.
+We're talking about listening to
+[`pointerup`](/web/updates/2016/10/pointer-events), `click`, and `touchend`
+events. In other words, the user must click on a button before your web app can
+actually make some noise.
+
+    playButton.addEventListener('pointerup', function(event) {
+      let audio = document.querySelector('audio');
+
+      // User interacted with the page. Let's play audio...
+      audio.play()
+      .then(_ => { /* Set up Media Session... */ })
+      .catch(error => { console.log(error) });
+    });
+
+Note: If the `<audio>` element has the `controls` attribute, you can simply set
+up the Media Session in the audio `play` event listener instead which occurs
```

Comment:
The rules are these:
* When referring to the API proper, capitalize. Media Session API
* When referring to a generic media session, do not capitalize. media session
* When referring to an instance of the `MediaSession` interface, like this: `mediaSession` (or something similar)
* When referring to the `MediaSession` interface itself, like this: `MediaSession`

Please verify throughout.